eliot/_action.py

```diff
@@ -94,7 +104,7 @@ def fromString(cls, string):
 
         @return: L{TaskLevel} parsed from the string.
         """
-        return cls(level=[int(i) for i in string.split("/") if i])
+        return cls(level=pvector([int(i) for i in string.split("/") if i]))
```

Comment:
Not actionable, but you don't need need the `pvector`; `pvector_field` will do taht for you).